fava/help/filters.md

```diff
@@ -30,14 +30,13 @@ the account name, or a regular expression matching the account name, e.g.
 This final filter allows you to filter entries by various attributes.
 
 -   Filter by `#tag` or `^link`.
--   Filter by payee `payee:".*restaurant.*"`, or narration `narration:'Dinner with Joe'`,
-    or any other entry attribute. The argument needs to be quoted (with `'` or
-    `"`) if it contains spaces and can either be simply a string or a regular
-    expression that matches the attribute. The regular expression matching is
-    case insensitive.
--   Search payee and narration by supplying a simple string, which again is
-    matched as a regular expression. For Note directives, the comment will be
-    searched.
+-   Filter by any entry attribute, such as payee `payee:".*restaurant.*"` or
```

Comment:
The `.*`s can be removed. Maybe you could add an example with an anchored regex?